frontend/webapp/cypress/e2e/05-actions.cy.ts

```diff
@@ -0,0 +1,103 @@
+import { ROUTES } from '../../utils/constants/routes';
+
+// The number of CRDs that exist in the cluster before running any tests should be 0.
+// Tests will fail if you have existing CRDs in the cluster.
+// If you have to run tests locally, make sure to clean up the cluster before running the tests.
+
+describe('Actions CRUD', () => {
+  const namespace = 'odigos-system';
```

Comment:
this can be reuse from a common file to other tests
```suggestion
  const namespace = 'odigos-system';
```